lib/put.js

```diff
@@ -44,6 +44,9 @@ PutRequest.prototype._finalize = function () {
     clock: this._clock
   }
 
+  // TODO: allow to set this flag explicitly
+  if (node.value === null) node.deleted = true
```

Comment:
Still want this one in there?

Reply:
Ah yea for now I need to keep it as that's how deletes are implemented internally (simply as `db.put(key, null)`). We can rewrite that at some point though

Reply:
Yea they'll turn into a delete (but we'll fix that asap)